eng/common/scripts/Invoke-GitHub-API.ps1

```diff
@@ -0,0 +1,220 @@
+$GithubAPIBaseURI = "https://api.github.com/repos"
+
+function Get-Headers ($token) {
+  $headers = @{
+    Authorization = "bearer $token"
+  }
+  return $headers
```

Comment:
For auth we should just use the -Token parameter and -Authentication parameter.

Reply:
Wes had a question about that here https://github.com/Azure/azure-sdk-tools/pull/1081#discussion_r502647956